doc_source/calico.md

```diff
@@ -4,12 +4,34 @@
 
 **To install Calico on your Amazon EKS cluster**
 
-1. Apply the Calico manifest from the [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\. This manifest creates daemon sets in the `kube-system` namespace\.
+1. Apply the Calico manifest from the [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\.
+   This manifest creates the calico-node `DaemonSet` in the `kube-system` namespace\.
 
    ```
-   kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.0.0/config/v1.0/aws-k8s-cni-calico.yaml
+   kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.0.1/config/v1.0/calico.yaml
    ```
 
+**To upgrade Calico on your Amazon `v1.0.0` EKS cluster**
+
+1. Apply the EKS CNI manifest [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\.
+   This manifest creates the container networking interface (CNI) `DaemonSet` in the `kube-system` namespace\.
+
+   ```
+   kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.0.1/config/v1.0/aws-k8s-cni.yaml
+   ```
+
+1. Apply the Calico manifest from the [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\.
+   This manifest creates the calico-node `DaemonSet` in the `kube-system` namespace\.
+
+   ```
+   kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.0.1/config/v1.0/calico.yaml
+   ```
+
+> **Note**: When upgrading from an EKS v1.0.0 cluster running Calico v3.0.6, restart each `Pod` after applying
+  the EKS CNI manifest and the Calico manifest.
+
+**To verify Calico is running on your system**
+
 1. Watch the `kube-system` daemon sets and wait for the `aws-node` and `calico-node` daemon sets to have the `DESIRED` number of pods in the `READY` state\. When this happens, Calico is working\.
```

Comment:
I would suggest deleting the heading in line 33 and copying lines 35-47 and pasting them at the end of the install (line 13). So instead of creating a new heading here, you could conclude the install and upgrade with this step. I think it will be a little easier to follow.

Reply:
will do